pom.xml

```diff
@@ -32,6 +32,10 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
```

Comment:
I don't think we should change the parent pom, if we need this dependency we can add it to the child pom. Maybe if you add it in web-ui.pom it will be better?

Reply:
I agree, i moved this dependency